docs/user-guide/release-notes/2.11.x.rst

```diff
@@ -2,6 +2,16 @@
 Pulp 2.11 Release Notes
 =======================
 
+EL6 Support Changes
+===================
+
+The 2.11 release is the last release stream with support for EL6. In preparation for 2.12+, EL6
```

Comment:
It would be valuable to clarify that "support" is for deploying pulp itself. This does not apply to where pulp-agent can run, whether pulp can sync and publish EL6 repos, etc.